2024/04/14 20:05:39 [emerg] 29400#25032: CreateDirectory() "D:\codes_dc\comsys\nginx/temp/client_body_temp" failed (3: The system cannot find the path specified)
2024/04/14 20:05:56 [error] 4280#2624: *1 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:05:56 [error] 4280#2624: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:06:56 [error] 27108#24572: *1 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:07:49 [error] 29044#29604: *1 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:07:49 [error] 29044#29604: *1 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:08:43 [emerg] 16992#27724: CreateFile() "D:\codes_dc\comsys\nginx/conf/nginx.conf" failed (3: The system cannot find the path specified)
2024/04/14 20:08:53 [notice] 28964#25448: signal process started
2024/04/14 20:08:55 [error] 26756#10168: *2 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:08:55 [error] 26756#10168: *2 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:09:18 [error] 26756#10168: *2 CreateFile() "./../dist/index.html" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /index.html HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:09:21 [error] 26756#10168: *2 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:10:02 [error] 26756#10168: *3 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:10:34 [error] 26756#10168: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://[::1]:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:10:34 [error] 26756#10168: *2 upstream sent no valid HTTP/1.0 header while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:10:36 [error] 26756#10168: *3 upstream sent no valid HTTP/1.0 header while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:10:49 [error] 26756#10168: *4 upstream sent no valid HTTP/1.0 header while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:13:07 [notice] 25424#28592: signal process started
2024/04/14 20:13:09 [error] 29672#17284: *9 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:13:12 [error] 29672#17284: *11 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:13:16 [error] 29672#17284: *15 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:13:17 [error] 29672#17284: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://[::1]:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:13:17 [error] 29672#17284: *11 upstream sent no valid HTTP/1.0 header while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:14:04 [notice] 24456#28204: signal process started
2024/04/14 20:14:07 [error] 26476#2624: *19 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:14:07 [error] 26476#2624: *17 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:14:09 [error] 26476#2624: *17 upstream sent no valid HTTP/1.0 header while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Auth/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/api/auth.Auth/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:16:37 [notice] 17396#21364: signal process started
2024/04/14 20:16:41 [error] 25512#24268: *24 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:16:41 [error] 25512#24268: *21 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:17:48 [error] 25512#24268: *30 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:17:48 [error] 25512#24268: *28 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:20:11 [notice] 27984#28228: signal process started
2024/04/14 20:20:15 [error] 3092#29672: *35 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:20:15 [error] 3092#29672: *32 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:20:40 [error] 3092#29672: *35 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:21:24 [error] 3092#29672: *42 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:21:25 [error] 3092#29672: *40 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:22:15 [error] 3092#29672: *48 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:22:16 [error] 3092#29672: *47 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:26:59 [error] 3092#29672: *54 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:26:59 [error] 3092#29672: *52 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:28:06 [error] 3092#29672: *60 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:28:06 [error] 3092#29672: *57 CreateFile() "./../dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/14 20:39:55 [error] 3092#29672: *63 CreateFile() "./../dist/chebupela" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /chebupela HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:39:57 [error] 3092#29672: *64 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:40:22 [notice] 21564#11620: signal process started
2024/04/14 20:40:23 [error] 18944#25108: *65 "D:\codes_dc\comsys\nginx/html/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:40:25 [error] 18944#25108: *65 "D:\codes_dc\comsys\nginx/html/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:41:32 [error] 18944#25108: *65 CreateFile() "D:\codes_dc\comsys\nginx/html/index.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /index.html HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:41:32 [error] 18944#25108: *65 CreateFile() "D:\codes_dc\comsys\nginx/html/favicon.ico" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/index.html"
2024/04/14 20:41:45 [error] 18944#25108: *66 "D:\codes_dc\comsys\nginx/html/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:41:48 [error] 18944#25108: *66 "D:\codes_dc\comsys\nginx/html/index.html" is not found (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:41:51 [notice] 27604#12096: signal process started
2024/04/14 20:41:53 [error] 27908#9636: *69 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:44:04 [error] 27908#9636: *71 CreateFile() "./../dist/comsys-wasm/index.scss" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: comsys, request: "GET /comsys-wasm/index.scss HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:44:08 [error] 27908#9636: *72 CreateFile() "./../dist/u" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: comsys, request: "GET /u HTTP/1.1", host: "127.0.0.1"
2024/04/14 20:44:43 [notice] 26084#3180: signal process started
2024/04/15 20:46:20 [error] 5504#18968: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/16 01:16:54 [error] 5504#18968: *30 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/16 22:06:25 [error] 1424#7716: *10 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "OPTIONS /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "http://127.0.0.1:88/"
2024/04/16 22:06:53 [error] 1424#7716: *12 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 16:57:40 [error] 14744#3020: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 16:57:42 [error] 14744#3020: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:26:57 [error] 14744#3020: *9 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:26:59 [error] 14744#3020: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:01 [error] 14744#3020: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:04 [error] 14744#3020: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:05 [error] 14744#3020: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:06 [error] 14744#3020: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:06 [error] 14744#3020: *16 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:27:17 [error] 14744#3020: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:30:40 [error] 14744#3020: *24 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:32:46 [error] 14744#3020: *29 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:33:20 [error] 14744#3020: *32 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:34:04 [error] 14744#3020: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:37:00 [error] 14744#3020: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:43:34 [error] 14744#3020: *46 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:43:38 [error] 14744#3020: *46 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:45:00 [error] 14744#3020: *51 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:45:22 [error] 14744#3020: *55 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:45:39 [error] 14744#3020: *59 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:46:32 [error] 14744#3020: *59 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:46:34 [error] 14744#3020: *59 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:47:03 [error] 14744#3020: *65 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:49:20 [error] 14744#3020: *70 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:49:38 [error] 14744#3020: *74 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:50:03 [error] 14744#3020: *77 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:50:05 [error] 14744#3020: *82 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:51:36 [error] 14744#3020: *89 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:51:38 [error] 14744#3020: *92 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:51:57 [error] 14744#3020: *92 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:52:40 [error] 14744#3020: *98 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:54:53 [error] 14744#3020: *102 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:54:58 [error] 14744#3020: *102 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:55:11 [error] 14744#3020: *108 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:55:17 [error] 14744#3020: *108 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:56:12 [error] 14744#3020: *113 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:57:02 [error] 14744#3020: *117 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:57:37 [error] 14744#3020: *121 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:57:39 [error] 14744#3020: *124 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:57:43 [error] 14744#3020: *124 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 18:58:16 [error] 14744#3020: *130 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:00:11 [error] 14744#3020: *133 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:02:00 [error] 14744#3020: *138 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:02:04 [error] 14744#3020: *138 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:02:07 [error] 14744#3020: *138 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:08:44 [error] 14744#3020: *143 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:09:01 [error] 14744#3020: *143 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:09:44 [error] 14744#3020: *149 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:10:23 [error] 14744#3020: *148 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:10:27 [error] 14744#3020: *148 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:11:02 [error] 14744#3020: *155 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:11:49 [error] 14744#3020: *177 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:14:16 [error] 14744#3020: *183 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:14:42 [error] 14744#3020: *187 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:16:09 [error] 14744#3020: *191 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:16:14 [error] 14744#3020: *191 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:16:35 [error] 14744#3020: *196 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:19:15 [error] 14744#3020: *200 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:19:19 [error] 14744#3020: *200 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:20:26 [error] 14744#3020: *204 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:20:38 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:20:42 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:20:45 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:21:16 [error] 14744#3020: *207 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:21:19 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:21:56 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:22:00 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:22:05 [error] 14744#3020: *208 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:24:46 [error] 14744#3020: *223 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:25:41 [error] 14744#3020: *227 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:39:44 [error] 14744#3020: *233 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:42:11 [error] 14744#3020: *237 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:43:15 [error] 14744#3020: *241 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:43:51 [error] 14744#3020: *240 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:44:12 [error] 14744#3020: *246 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:44:35 [error] 14744#3020: *249 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:44:51 [error] 14744#3020: *250 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:45:56 [error] 14744#3020: *256 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:46:08 [error] 14744#3020: *256 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:46:33 [error] 14744#3020: *260 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:46:42 [error] 14744#3020: *264 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:46:49 [error] 14744#3020: *264 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:47:00 [error] 14744#3020: *271 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:47:29 [error] 14744#3020: *271 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 19:48:37 [error] 14744#3020: *279 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:00:45 [error] 14744#3020: *285 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:00:50 [error] 14744#3020: *285 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:04:09 [error] 14744#3020: *288 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:04:49 [error] 14744#3020: *292 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:05:16 [error] 14744#3020: *292 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:05:45 [error] 14744#3020: *298 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:06:39 [error] 14744#3020: *298 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:07:49 [error] 14744#3020: *304 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:08:55 [error] 14744#3020: *304 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:10:17 [error] 14744#3020: *315 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "OPTIONS /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://172.30.32.1/"
2024/04/17 20:12:11 [error] 14744#3020: *315 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "OPTIONS /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://172.30.32.1/"
2024/04/17 20:16:40 [error] 14744#3020: *323 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "OPTIONS /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://192.168.88.209/"
2024/04/17 20:32:01 [error] 14744#3020: *329 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 20:32:30 [error] 14744#3020: *330 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:17:16 [error] 14744#3020: *336 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:47:23 [error] 14744#3020: *340 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:47:53 [error] 14744#3020: *340 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:49:40 [error] 14744#3020: *339 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:51:21 [error] 14744#3020: *339 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 21:53:12 [error] 14744#3020: *339 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:01:02 [error] 14744#3020: *351 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:03:39 [error] 14744#3020: *354 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:03:57 [error] 14744#3020: *354 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:04:00 [error] 14744#3020: *358 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:05:52 [error] 14744#3020: *358 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:09:17 [error] 14744#3020: *365 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:23:21 [error] 14744#3020: *369 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:24:19 [error] 14744#3020: *370 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:24:51 [error] 14744#3020: *373 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:25:49 [error] 14744#3020: *373 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:26:18 [error] 14744#3020: *374 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:27:09 [error] 14744#3020: *380 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:27:27 [error] 14744#3020: *379 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:28:45 [error] 14744#3020: *380 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:30:20 [error] 14744#3020: *379 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:33:53 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:34:21 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:35:08 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:36:28 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:36:54 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:37:31 [error] 14744#3020: *391 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:38:05 [error] 14744#3020: *389 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:38:19 [error] 14744#3020: *399 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:38:55 [error] 14744#3020: *398 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:39:38 [error] 14744#3020: *399 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:43:18 [error] 14744#3020: *406 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:43:20 [error] 14744#3020: *408 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:55:34 [error] 14744#3020: *411 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 22:55:36 [error] 14744#3020: *415 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:07:44 [error] 14744#3020: *431 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:08:05 [error] 14744#3020: *431 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:14:43 [error] 14744#3020: *441 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:24:13 [error] 14744#3020: *445 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:44:49 [error] 14744#3020: *454 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:44:57 [error] 14744#3020: *454 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:45:03 [error] 14744#3020: *454 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "GET /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1"
2024/04/17 23:49:26 [error] 14744#3020: *478 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/17 23:49:29 [error] 14744#3020: *480 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/18 02:36:55 [error] 14744#3020: *511 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/18 14:50:58 [error] 26128#12156: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/18 20:24:55 [error] 26128#12156: *136 directory index of "./../dist/" is forbidden, client: 127.0.0.1, server: comsys, request: "GET / HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/19 18:28:27 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:28:31 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:28:32 [error] 18284#18612: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:28:32 [error] 18284#18612: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:28:33 [error] 18284#18612: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:28:45 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:29:22 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:29:27 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:29:32 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/19 18:29:37 [error] 18284#18612: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.88.209, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "192.168.88.209", referrer: "https://192.168.88.209/"
2024/04/20 15:53:57 [error] 9584#4256: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:53:59 [error] 9584#4256: *5 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:02 [error] 9584#4256: *5 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:03 [error] 9584#4256: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:04 [error] 9584#4256: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:04 [error] 9584#4256: *12 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:04 [error] 9584#4256: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/20 15:54:04 [error] 9584#4256: *5 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 19:55:37 [error] 24072#10308: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 19:56:23 [error] 24072#10308: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/aby"
2024/04/29 20:14:30 [error] 24072#10308: *9 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 20:16:07 [error] 24072#10308: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 20:16:07 [error] 24072#10308: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 20:16:12 [error] 24072#10308: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 22:25:00 [error] 24072#10308: *22 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/04/29 22:25:13 [error] 24072#10308: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:48:08 [error] 12568#2340: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/01 11:48:08 [error] 12568#2340: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/01 11:51:47 [error] 12568#2340: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/01 11:51:47 [error] 12568#2340: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/01 11:52:44 [error] 12568#2340: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:44 [error] 12568#2340: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:44 [error] 12568#2340: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:48 [error] 12568#2340: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:48 [error] 12568#2340: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:48 [error] 12568#2340: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:52:51 [error] 12568#2340: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/01 11:55:12 [error] 12568#2340: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:53 [error] 26520#40424: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:53 [error] 26520#40424: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:54 [error] 26520#40424: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:54 [error] 26520#40424: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:56 [error] 26520#40424: *9 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:56 [error] 26520#40424: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:57 [error] 26520#40424: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:57 [error] 26520#40424: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:58 [error] 26520#40424: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:53:58 [error] 26520#40424: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/02 22:54:00 [error] 26520#40424: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/03 17:55:21 [error] 12740#2916: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 17:55:21 [error] 12740#2916: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:17 [error] 12740#2916: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:17 [error] 12740#2916: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:17 [error] 12740#2916: *15 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /webapp/comps/new HTTP/1.1", host: "127.0.0.1"
2024/05/03 18:00:18 [error] 12740#2916: *10 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:18 [error] 12740#2916: *12 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:21 [error] 12740#2916: *16 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:00:21 [error] 12740#2916: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 18:06:52 [error] 12740#2916: *72 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /webapp/comps/new HTTP/1.1", host: "127.0.0.1"
2024/05/03 22:20:58 [error] 12740#2916: *158 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 22:20:58 [error] 12740#2916: *156 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 22:56:40 [error] 12740#2916: *162 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/03 22:56:40 [error] 12740#2916: *161 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/06 23:29:16 [error] 54904#41760: *54 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/06 23:29:16 [error] 54904#41760: *53 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/06 23:29:22 [error] 54904#41760: *54 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/06 23:29:22 [error] 54904#41760: *53 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/12 17:27:12 [error] 21748#32172: *5 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/comps/"
2024/05/16 17:45:32 [error] 13800#28452: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/17 21:06:24 [error] 14304#19504: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/17 21:06:29 [error] 14304#19504: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/17 21:06:35 [error] 14304#19504: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/17 21:06:38 [error] 14304#19504: *3 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/05/17 21:44:35 [error] 14304#19504: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/18 23:52:01 [error] 14632#24868: *20 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/19 00:33:55 [error] 14632#24868: *35 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/DeclareCompetition HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/DeclareCompetition", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/05/31 20:51:11 [error] 26872#19800: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetComps HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetComps", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/05/31 20:51:20 [error] 26872#19800: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetComps HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetComps", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/05/31 20:51:24 [error] 26872#19800: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetComps HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetComps", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/16 21:08:10 [error] 20908#19412: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/drop/12"
2024/07/16 21:31:05 [error] 20908#19412: *5 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 21:31:05 [error] 20908#19412: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetComps HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetComps", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 21:55:32 [error] 20908#19412: *9 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 21:55:32 [error] 20908#19412: *11 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetComps HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetComps", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 22:07:16 [error] 20908#19412: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 22:07:31 [error] 20908#19412: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/16 22:11:11 [error] 20908#19412: *36 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/12"
2024/07/25 18:21:22 [error] 11648#13516: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/new"
2024/07/25 18:21:38 [error] 11648#13516: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/users/new"
2024/07/25 18:42:55 [error] 11648#13516: *10 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:43:21 [error] 11648#13516: *10 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:44:47 [error] 11648#13516: *10 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:49:49 [error] 11648#13516: *15 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:52:09 [error] 11648#13516: *18 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:52:17 [error] 11648#13516: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:52:21 [error] 11648#13516: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:52:46 [error] 11648#13516: *21 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:52:47 [error] 11648#13516: *20 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:53:03 [error] 11648#13516: *28 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:55:49 [error] 11648#13516: *33 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:56:23 [error] 11648#13516: *33 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:56:55 [error] 11648#13516: *33 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 18:58:12 [error] 11648#13516: *32 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:35 [error] 11648#13516: *39 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:55 [error] 11648#13516: *39 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:56 [error] 11648#13516: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:56 [error] 11648#13516: *40 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:56 [error] 11648#13516: *38 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:03:57 [error] 11648#13516: *49 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:04:13 [error] 11648#13516: *39 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:04:15 [error] 11648#13516: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/25 19:04:15 [error] 11648#13516: *39 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/27 00:17:00 [error] 20136#19548: *4 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/28 17:38:22 [error] 8640#2652: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/28 17:38:28 [error] 8640#2652: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/28 17:38:37 [error] 8640#2652: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/28 17:39:15 [emerg] 6072#13448: CreateFile() "D:\codes_dc\comsys\nginx/.\nginx\ngnix.conf" failed (3: The system cannot find the path specified)
2024/07/28 17:39:33 [error] 11732#16100: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:41:15 [error] 21240#6632: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:44:05 [error] 21240#6632: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:44:21 [error] 21240#6632: *6 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:44:23 [error] 21240#6632: *8 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Authorize HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Authorize", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:47:18 [error] 21240#6632: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/29 14:47:31 [error] 21240#6632: *13 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/Registration HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/Registration", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/07/30 20:28:23 [error] 24296#4484: *20 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/30 20:28:23 [error] 24296#4484: *18 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/31 12:21:49 [error] 4984#20664: *4 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/31 12:23:07 [error] 4984#20664: *4 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/31 12:36:09 [error] 4984#20664: *15 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/07/31 14:07:26 [error] 4984#20664: *42 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /webapp/comps/mod/1 HTTP/1.1", host: "127.0.0.1"
2024/07/31 14:37:56 [error] 4984#20664: *57 WSASend() failed (10054: An existing connection was forcibly closed by the remote host) while sending request to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompDeclaration HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompDeclaration", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/08/04 21:56:15 [crit] 12356#12556: *39 GetFileAttributesEx() "./../dist/index.html" failed (5: Access is denied), client: 127.0.0.1, server: comsys, request: "GET /webapp/comps/mod/1 HTTP/1.1", host: "127.0.0.1"
2024/08/04 21:56:15 [error] 12356#12556: *39 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /webapp/comps/mod/1 HTTP/1.1", host: "127.0.0.1"
2024/08/04 21:57:29 [error] 12356#12556: *51 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 127.0.0.1, server: comsys, request: "GET /consys-wasm-3f1d55d5b1bf070f.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps/mod/1"
2024/08/06 15:40:55 [error] 15680#25520: *19 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompsIds HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompsIds", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps"
2024/08/06 15:41:01 [error] 15680#25520: *19 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompsIds HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompsIds", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps"
2024/08/06 15:44:12 [error] 15680#25520: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompsIds HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompsIds", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps"
2024/08/06 21:22:31 [error] 15680#25520: *36 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps"
2024/08/06 21:22:31 [error] 15680#25520: *38 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp.CompetitionDeclarator/GetCompsIds HTTP/1.1", upstream: "http://127.0.0.1:50051/comp.CompetitionDeclarator/GetCompsIds", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/comps"
2024/08/13 22:24:32 [error] 896#23988: *21 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:24:32 [error] 896#23988: *16 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:24:32 [error] 896#23988: *23 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:27:18 [error] 896#23988: *48 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:27:18 [error] 896#23988: *29 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:27:18 [error] 896#23988: *49 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:29:57 [error] 896#23988: *70 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:29:57 [error] 896#23988: *47 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:39:12 [error] 896#23988: *92 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:43:03 [error] 896#23988: *101 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:53:12 [error] 4216#7336: *3 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:53:12 [error] 4216#7336: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:54:35 [error] 4216#7336: *5 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 22:59:06 [error] 4216#7336: *26 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:05:06 [error] 4216#7336: *36 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:07:03 [error] 4216#7336: *44 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:07:19 [error] 4216#7336: *47 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:08:32 [error] 4216#7336: *47 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:11:58 [error] 4216#7336: *56 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:12:03 [error] 4216#7336: *59 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:13:43 [error] 4216#7336: *62 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:46:29 [error] 4216#7336: *73 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:48:39 [error] 4216#7336: *83 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/13 23:50:31 [error] 4216#7336: *93 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:24:38 [error] 8508#18836: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:26:18 [error] 8508#18836: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:26:42 [error] 8508#18836: *1 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/Run HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/Run", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:28:38 [error] 8508#18836: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:31:13 [error] 8508#18836: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:49:42 [error] 8508#18836: *28 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 13:53:44 [error] 8508#18836: *40 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 14:30:10 [error] 8508#18836: *52 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 14:44:20 [error] 8508#18836: *62 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 14:51:22 [error] 8508#18836: *74 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 14:51:22 [error] 8508#18836: *75 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 15:01:21 [error] 8508#18836: *109 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 15:06:48 [error] 8508#18836: *116 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 15:08:25 [error] 8508#18836: *114 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 19:26:54 [error] 8508#18836: *161 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 19:30:10 [error] 8508#18836: *168 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:09:17 [error] 8508#18836: *176 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:15:45 [error] 8508#18836: *176 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:21:50 [error] 8508#18836: *192 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:22:09 [error] 8508#18836: *201 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:34:24 [error] 8508#18836: *206 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:34:24 [error] 8508#18836: *213 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 20:36:49 [error] 8508#18836: *224 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 21:12:04 [error] 8508#18836: *278 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 21:41:47 [error] 8508#18836: *288 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/14 22:03:11 [error] 8508#18836: *294 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:06:11 [error] 12444#17740: *2 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:06:11 [error] 12444#17740: *9 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:06:22 [error] 12444#17740: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:06:25 [error] 12444#17740: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:06:34 [error] 12444#17740: *14 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:45:05 [error] 12444#17740: *20 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 13:56:20 [emerg] 12052#6436: unexpected "}" in D:\codes_dc\comsys\nginx/ngnix.conf:65
2024/08/15 13:56:54 [emerg] 9084#3272: directive "proxy_pass" is not terminated by ";" in D:\codes_dc\comsys\nginx/ngnix.conf:65
2024/08/15 13:56:58 [emerg] 2788#21104: unexpected "}" in D:\codes_dc\comsys\nginx/ngnix.conf:64
2024/08/15 16:23:03 [error] 17432#12284: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: comsys, request: "POST /api/auth.Authentication/GetAuth HTTP/1.1", upstream: "http://127.0.0.1:50051/auth.Authentication/GetAuth", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2024/08/15 16:41:59 [error] 17432#12284: *2 WSARecv() failed (10054: An existing connection was forcibly closed by the remote host) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
2024/08/15 16:46:14 [error] 17432#12284: *14 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while reading upstream, client: 127.0.0.1, server: comsys, request: "POST /api/comp_handler.CompetitionHandler/startEQMessageStream HTTP/1.1", upstream: "http://127.0.0.1:50051/comp_handler.CompetitionHandler/startEQMessageStream", host: "127.0.0.1", referrer: "https://127.0.0.1/webapp/testing"
